sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_pipeline_component_builder.py

```diff
@@ -381,7 +388,7 @@ def _get_name_or_component_name(node: Union[BaseNode, AutoMLJob]):
         return result
 
 
-def _build_pipeline_parameter(func, kwargs=None, non_pipeline_parameter_dict=None):
+def _build_pipeline_parameter(func, original_kwargs, group_default_kwargs=None, non_pipeline_parameter_dict=None):
```

Comment:
Add a '*' here, as all the parameters are dict type

Reply:
Updated